common/core/web/tools/sentry-manager/src/index.ts

```diff
@@ -7,21 +7,24 @@ namespace com.keyman {
    */
   let DEBUG = false;
 
-  type Options = typeof KeymanSentryManager.DEFAULT_OPTIONS;
+  type Options = {
```

Comment:
Any thoughts if this goes in common/core/web/utils/src/deviceSpecs.ts

Reply:
It does not; the separation is very intentional.

The file you mention is meant to abstract away the actual device for testing rules that don't match the actual device that unit tests are run upon.

Also, note that KMW's device detection - even the full-fledged `Device` class - doesn't actually distinguish between 'native' iOS (using keymanweb.com from Safari) and 'embedded' iOS (the Keyman app).  `keyman.isEmbedded` (on DOM-KMW's `KeymanBase`) supplies the missing piece of information needed for auto-detection... but we really, _really_ don't want to compile `sentry-manager` against that monolith.

Technically, we _could_ use a `@ts-ignore` to make auto-detection possible anyway, but this would also lose us the ability to distinguish between host sites if we wanted it.  For example, we could add `keymanweb.com` and `help.keyman.com` (for KMW's guide pages) as "host platforms" if we wanted, providing extra context about exactly which uses of KMW are experiencing difficulties.